### PR TITLE
Removed Solution Activity Log from LAW

### PR DESCRIPTION
**Pull request by lachaves**
# Overview/Summary

Remove Activity Log solution from Bicep RI
Update and Bicep documentation referencing this

## This PR fixes/adds/changes/removes

1. PR #414
2.  Closes [AB#25141](https://dev.azure.com/CSUSolEng/2a9d5a5a-8998-4ad0-81c8-ef3045e4da97/_workitems/edit/25141)

### Breaking Changes

1. None

## Testing Evidence

Executed the deployment and Activity Log Solution is not deployed. 

## As part of this Pull Request I have

- [X ] Read the [Contribution Guide](https://github.com/Azure/ALZ-Bicep/wiki/Contributing) and ensured this PR is compliant with the guide
- [X ] Ensured the resource API versions in `.bicep` file/s I am adding/editing are using the latest API version possible
- [X ] Checked for duplicate [Pull Requests](https://github.com/Azure/ALZ-Bicep/pulls)
- [X ] Associated it with relevant [GitHub Issues](https://github.com/Azure/ALZ-Bicep/issues)
- [X ] *(ALZ Bicep Core Team Only)* Associated it with relevant [ADO Items](https://aka.ms/alz/bicep/backlog) - [AB#25141](https://dev.azure.com/CSUSolEng/2a9d5a5a-8998-4ad0-81c8-ef3045e4da97/_workitems/edit/25141)
- [ X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/ALZ-Bicep/tree/main)
- [X ] Performed testing and provided evidence.
- [ ] Updated one or more of the following tests *(if required)*
  - [Unit](https://github.com/Azure/ALZ-Bicep/blob/main/.github/workflows/bicep-build-to-validate.yml)
  - [Linting](https://github.com/Azure/ALZ-Bicep/tree/main/.github/workflows)
  - [E2E (End-To-End)](https://github.com/Azure/ALZ-Bicep/blob/main/tests/pipelines/bicep-build-to-validate.yml)
  - [ValidateAzCloud (Base validation in Azure Cloud)](https://github.com/Azure/ALZ-Bicep/blob/main/tests/pipelines/base-unit-validate.yml)
  - [ValidateMcCloud (Base validation in Azure China Cloud)](https://github.com/Azure/ALZ-Bicep/blob/main/tests/pipelines/mc-base-unit-validate.yml)
- [X ] Updated relevant and associated documentation (e.g. Contribution Guide, Module READMEs, Wiki Docs etc.)
- [ ] If relevant, created or updated Code Tours [here](https://github.com/Azure/ALZ-Bicep/blob/main/.vscode/tours)

### CODE REVIEW

File: infra-as-code/bicep/modules/logging/logging.bicep [hunk 1]
```diff
@@ -39,11 +38,10 @@ param parLogAnalyticsWorkspaceLogRetentionInDays int = 365
   'Updates'
   'VMInsights'
 ])
-@sys.description('Solutions that will be added to the Log Analytics Workspace. Default: [AgentHealthAssessment, AntiMalware, AzureActivity, ChangeTracking, Security, SecurityInsights, ServiceMap, SQLAssessment, Updates, VMInsights]')
+@sys.description('Solutions that will be added to the Log Analytics Workspace. Default: [AgentHealthAssessment, AntiMalware, ChangeTracking, Security, SecurityInsights, ServiceMap, SQLAssessment, Updates, VMInsights]')
```
Comment (JamJarchitect) [inline infra-as-code/bicep/modules/logging/logging.bicep:41]: Need to remove the `Default:` section in each of the parameters description in this PR.

When the parameter markdowns are generated it displays the default value underneath. Other modules to be verified for same parameter description `Default:` detail.

**Review by JamJarchitect — CHANGES_REQUESTED** [review]
This looks good, just need to remove detail from parameter descriptions to start a standardisation!

**Review by JamJarchitect — APPROVED** [review]
LGTM

**Review by jtracey93 — APPROVED** [review]

**Pull request merged**